styles.css

```diff
@@ -6,108 +6,479 @@
 }
 
 body {
```

Comment:
- [ ] Kindly make sure that your design matches the Figma template as much as possible (e.g., font, colors, images, tests, margins), and is responsive. You can use [CSS Media Queries](https://www.w3schools.com/css/css3_mediaqueries.asp) to make your site responsive, and favor using units that are percent base such as (vw, vh, %) instead of `px` when adding width, height and margins as they make your page more responsive.